### ISSUE TITLE
Dist archives for releases

### ISSUE THREAD

**Issue by Dridi** [acrisci/playerctl#83]
Hello,

Playerctl is filling a gap that has been itching me for years now, and on the first try it JustWorked(tm). Thanks a lot for that.

However, I tend to avoid installing software manually and I would like to package this in Fedora, so in order to reduce build-time dependencies it would be great if you could publish a `make dist` archive alongside the deb and rpm files you upload with releases.

Ideally latest release and future releases.

I'm working on a spec for Fedora based on this assumption.

Could relate to #42.

Cheers

**Comment by Dridi** [comment]
I guess I'll have to learn meson basics :)

If there is no difference between a git checkout and a "dist" archive with meson, then it may not be necessary.

**Comment by Dridi** [comment]
I uploaded a spec file [1] that should pass a package review for Fedora from a dist archive [2] I built on my machine with `make dist`.

As soon as you attach a similar archive to the github release, I can kick off a package submission. Next release I'll figure how to update the spec to build playerctl with meson.

Thanks!

[1] https://dridi.fedorapeople.org/review/playerctl.spec
[2] https://dridi.fedorapeople.org/review/playerctl-0.6.1.tar.gz
